Integrations/integration-MaxMind_GeoIP2.yml

```diff
@@ -0,0 +1,293 @@
+                hr['User Type'] = traits['user_type']
+                maxmind_ec['UserType'] = traits['user_type']
+            if 'domain' in traits:
+                hr['Domain'] = traits['domain']
+                maxmind_ec['Domain'] = traits['domain']
+            if 'is_anonymous' in traits:
+                hr['Is Anonymous'] = traits['is_anonymous']
+                maxmind_ec['Anonymous'] = traits['is_anonymous']
+            if 'is_hosting_provider' in traits:
+                hr['Is Hosting Provider'] = traits['is_hosting_provider']
+                maxmind_ec['Host'] = traits['is_hosting_provider']
+            if 'is_tor_exit_node' in traits:
+                hr['Is TOR Exit Node'] = traits['is_tor_exit_node']
+                maxmind_ec['Tor'] = traits['is_tor_exit_node']
+            if 'autonomous_system_number' in traits:
+                hr['ASN'] = traits['autonomous_system_number']
+                ip_ec['ASN'] = traits['autonomous_system_number']
+                maxmind_ec['ASN'] = traits['autonomous_system_number']
+            if 'autonomous_system_organization' in traits:
+                hr['Organization'] = traits['autonomous_system_organization']
+                maxmind_ec['Organization'] = traits['autonomous_system_organization']
+            hr['IP'] = traits['ip_address']
+            ip_ec['Address'] = traits['ip_address']
+            maxmind_ec['Address'] = traits['ip_address']
+            if 'isp' in traits:
+                hr['ISP'] = traits['isp']
+                maxmind_ec['ISP'] = traits['isp']
+        return hr, ip_ec, maxmind_ec
+
+
+
+    ''' FUNCTIONS '''
+    def get_geo_ip(query):
+        raw = http_request(query)
+        res_json = raw.json()
+        return res_json
+
+
+    def geo_ip_command():
+        ip_query = demisto.args().get('ip')
+        res_json = get_geo_ip(ip_query)
+        hr, ip_ec, maxmind_ec = format_results(res_json)
+        ec = ({
+          'IP(val.Address && val.Address == obj.Address)': ip_ec,
+          'MaxMind(val.Address && val.Address == obj.Address)': maxmind_ec
+        })
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['markdown'],
+            'Contents': res_json,
+            'HumanReadable': tableToMarkdown('{} - Scan Results'.format(ip_query), hr, HR_HEADERS, removeNull=True),
+            'EntryContext': ec
+        })
+
+
+    ''' EXECUTION CODE '''
+    LOG('command is %s' % (demisto.command(), ))
+    try:
+        if demisto.command() == 'ip':
+            geo_ip_command()
+        if demisto.command() == 'test-module':
+            raw = http_request('8.8.8.8')
+            demisto.results('ok')
+    except Exception as e:
+        LOG(e)
+        LOG.print_log(False)
+        return_error(e.message)
+  type: python
+  commands:
+  - name: ip
+    arguments:
+    - name: ip
+      required: true
+      description: IP Address to be queried
+    outputs:
```

Comment:
Not sure if the order matters, but lets keep that ordered, IP related than MaxMind related